x-pack/legacy/plugins/siem/public/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -127,7 +131,7 @@ const ProviderContainer = styled.div<{ isDragging: boolean }>`
     ${isDragging &&
       `
       & {
-        z-index: ${theme.eui.euiZLevel9} !important;
+        z-index: 9999 !important;
```

Comment:
Any thoughts on how to better handle this situation? In this instance, the map tooltip `EuiPopover` is managed in the [maps code](https://github.com/elastic/kibana/blob/16f8b7ce6aa7dfd1193efb158c214cdbff1b265e/x-pack/legacy/plugins/maps/public/connected_components/map/mb/tooltip_control/tooltip_control.js#L340-L351), so we can't pass in our own `z-index`. When you don't specify your own `z-index`, the tooltip `EuiPopover` [does some calculation](https://github.com/elastic/eui/blob/master/src/components/popover/popover.tsx#L455-L462) to ensure it's under any flyout that would appear. This isn't necessary in this implementation as we have an outside click detector that'll close the popover.

The map tooltip `EuiPopover` ends up with the following `z-index`:

``` css
.euiPopover__panel.euiPopover__panel-isOpen {
    z-index: 9900 !important;
}
```